.changeset/odd-apples-explain.md

```diff
@@ -0,0 +1,5 @@
+---
+"@snapshot-labs/sx": minor
+---
+
+Use `address` instead of `signer` for methods of the ethereum-tx client
```

Comment:
```suggestion
Use `address` instead of `signer` for following methods in Starknet EthereumTx client: `estimateProposeFee`, `estimateVoteFee`, `estimateUpdateProposalFee`, `getProposeHash`, `getVoteHash`, `getUpdateProposalHash`.
```